src/main/java/mate/academy/hibernate/relations/util/HibernateUtil.java

```diff
@@ -1,9 +1,24 @@
 package mate.academy.hibernate.relations.util;
 
 import org.hibernate.SessionFactory;
+import org.hibernate.cfg.Configuration;
 
 public class HibernateUtil {
+    public static final String CAN_T_BUILD_SESSION_FACTORY_MSG = "Can't build session factory ";
+    private static final SessionFactory session = getInitSessionFactory();
+
+    public HibernateUtil() {
+    }
```

Comment:
The constructor should be private to prevent instantiation of this utility class. Consider changing the constructor to `private HibernateUtil() {}`.